.github/workflows/ci.yml

```diff
@@ -147,8 +147,9 @@ jobs:
       - name: Install .NET SDK
         uses: actions/setup-dotnet@v3
         with:
-          # .NET 5 is deprecated and removed from GitHub Actions, we need to manually install it
+          # .NET 3.1 and 5 are deprecated and removed from GitHub Actions, we need to manually install them
```

Comment:
Do we have a plan to stop supporting it in the future?

Reply:
We do have a deprecation policy of "LTS + 1", which would technically make .NET 5 obsolete. However, we haven't moved on that yet.